Modules/_ssl.c

```diff
@@ -5686,15 +5697,21 @@ _ssl_enum_certificates_impl(PyObject *module, const char *store_name)
     Py_XDECREF(keyusage);
     Py_XDECREF(tup);
 
-    /* CERT_CLOSE_STORE_FORCE_FLAG forces freeing of memory for all contexts
-       associated with the store, in this case our collection store and the
-       associated system stores. */
-    if (!CertCloseStore(hCollectionStore, CERT_CLOSE_STORE_FORCE_FLAG)) {
+    /* CertCloseStore() with any argument can't close handles together.
+       In case of using CERT_CLOSE_STORE_FORCE_FLAG (not recommended),
```

Comment:
The comment is out of date. You are referencing ``CERT_CLOSE_STORE_FORCE_FLAG`` but no longer using ``CERT_CLOSE_STORE_FORCE_FLAG`` as flag.